### DISCUSSION TITLE
Not seeing touchbar support?

### DISCUSSION THREAD

**Discussion by LDubya** [ueberdosis/tiptap#923] [General Questions]
Hello,

ProseMirror has touchbar support in their default demos. I've gone through the TipTap demos on your landing page and do not see touchbar support. Can you confirm that TipTap has touchbar support out of the box, the way ProseMirror does?

**Comment by hanspagel** [comment]
There is no API for the touchbar, so there’s nothing ProseMirror or tiptap can do to enable/disable/add/remove touchbar support. 😏

Both use contenteditable="true", which should be enough to show the touchbar. I have no idea why it works or doesn’t work.
